examples/04_New Models.ipynb

```diff
@@ -280,58 +254,89 @@
    "outputs": [],
    "source": [
     "m = ThrownObject()\n",
-    "save = m.simulate_to_threshold(print = True, save_freq=1, threshold_keys='impact', dt=0.1)"
+    "save = m.simulate_to_threshold(print=True, save_freq=1, threshold_keys='impact', dt=0.1)"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "__Note__: Because our model takes in no inputs, we have no need to actually define a future loading function! As a result, we are simply passing in an empty Input Container. However, for most models, there would be inputs, thus a need for a future loading function. For more information on future loading functions and when to use them, please refer to the ProgPy [Future Loading](https://nasa.github.io/progpy/prog_models_guide.html#future-loading) Documentation."
+    "__Note__: Because our model takes in no inputs, we have no need to actually define a future loading function! As a result, we are simply passing in an empty Input Container. However, for most models, there would be inputs, thus a need for a future loading function. For more information on future loading functions and when to use them, please refer to the future loading section in 1. Simulation.\n",
```

Comment:
```suggestion
    "__Note__: Because our model takes in no inputs, we have no need to define a future loading function. However, for most models, there would be inputs, and thus a need for a future loading function. For more information on future loading functions and when to use them, please refer to the future loading section in 1. Simulation.\n",
```
We took the empty input container out (right?)